### PR TITLE
T21000 Add missing socs-soc-job-kernel-plan

### PR DESCRIPTION
**Pull request by gctucker**
Add the missing socs-soc-job-kernel-plan view to show a list of test
plan runs for a given kernel revision like the equivalent Tests view
but limited to one machine type (SoC).

Compared to the Tests view, simplify the regression information by not
including links to last passing / first failing kernel revisions since
this would be replaced with direct links to the failing test case
details anyway.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>

### CODE REVIEW

File: app/dashboard/static/js/app/view-socs-soc-job-kernel-plan.2020.5.js [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+            .done(getLabResultsDone);
+    }
+
+    function getRunsFailed() {
+        detailsFailed();
+        runsFailed();
+    }
+
+    function getRunsDone(response) {
+        console.log("OK");
```
Comment (alexandrasp) [inline app/dashboard/static/js/app/view-socs-soc-job-kernel-plan.2020.5.js]: In general looks good, but remove this `console.log` please.

Comment (gctucker) [reply app/dashboard/static/js/app/view-socs-soc-job-kernel-plan.2020.5.js]: Err, sure :innocent:

Comment (gctucker) [reply app/dashboard/static/js/app/view-socs-soc-job-kernel-plan.2020.5.js]: Fixed

**Review by alexandrasp — APPROVED** [review]
LGTM

**Comment by gctucker** [comment]
Tested OK on staging https://staging.kernelci.org/soc/rockchip/job/kernelci/kernel/staging-20200510.2/

**Comment by gctucker** [comment]
Actual example of this view: https://staging.kernelci.org/soc/rockchip/job/kernelci/kernel/staging-20200510.2/plan/baseline/

**Pull request merged**